docs-src/descriptions/current-in-streams.jade

```diff
@@ -31,21 +31,21 @@ pre(title='example')
     });
 
 p.
-  When a stream emits a value like this, it internally considered as a #[b current] value.
+  When a stream emits a value like this, it is internally considered as a #[b current] value.
   The #[a(href='#log') log] method will mark it as current,
-  and in #[a(href='#on-any') onAny] subscriber #[tt event.current] will equal #[b true].
+  and in #[a(href='#on-any') onAny] subscriber, #[tt event.current] will be equal to #[b true].
 
 p.
-  There is some issues with all this, but also some benefits.
+  There are some issues with all this, but also some benefits.
   Issue number one is that only the first subscriber gets this value (or error).
   Even if it was an #[b onEnd] subscriber, it'll still eat all current values,
   and further subscribers won't get them.
-  Second issue is that it may be considered as not semantically correct behaviour
-  i.e., the fact that moments when stream emits values depend on when it gets subscribers ...
+  Second issue is that it may be considered as a not semantically correct behaviour,
+  i.e., because the moment when the stream emits values depends on when it gets subscribers...
 
 p.
-  To benefits, first — it allows you to define current value in
-  #[a(href='#from-binder') fromBinder], it's easier to show in example:
+  First benefit — it allows you to define the current value in
+  #[a(href='#from-binder') fromBinder]. It is easier as shown in this example:
```

Comment:
In original sentence I meant that it harder to explain than to show in a example, but now it has different meaning.